httpgrpc/server/server.go

```diff
@@ -32,17 +32,31 @@ var (
 	DoNotLogErrorHeaderKey = http.CanonicalHeaderKey("X-DoNotLogError")
 )
 
+type Options func(*Server)
```

Comment:
This define a single option function:

```suggestion
type Option func(*Server)
```